ebs/gp2_gp3_migration.sh

```diff
@@ -0,0 +1,248 @@
+#! /bin/bash
+# Author : @razguru
+# Version : 1.1 - - Added cross account action through --profile assume role
+# Desc : Discover or/and modify GP2 volumes with/without snapshot
+
```

Comment:
The script needs the proper copyright and information about setting up and running AWS CLI.
See 
https://github.com/awsdocs/aws-doc-sdk-examples/blob/2ea3c65d947fbb43138bd44658ca03d0276e0b22/aws-cli/bash-linux/iam/iam_operations.sh#L3